### PR TITLE
Support multiple files in COMPOSE_FILE env var

### LINKED ISSUE #2395: Suggestion to use an env variable for -f file selection
The current option -f is great but error prone when deploying in production.

how about using a environment variable - something like 

$DOCKER-COMPOSER-ENV=prod

if not set  use docker-compose.yml and  show a warning no env set so using only the main file .....
if set use docker-compose.yml and extend using a file  based on the env 
docker-compose-$DOCKER-COMPOSER-ENV.yml 

$DOCKER-COMPOSER-ENV=prod
will run:
docker-compose -f docker-compose.yml -f docker-compose-prod.yml

### PR DESCRIPTION
**Pull request by dnephin**
Fixes #2395

This should make it easier to use multiple compose files when the files aren't the default.

### CODE REVIEW

File: compose/const.py [hunk 1]
```diff
@@ -15,6 +15,8 @@
 LABEL_VERSION = 'com.docker.compose.version'
 LABEL_CONFIG_HASH = 'com.docker.compose.config-hash'
 
+PATH_SEPARATOR = ';' if IS_WINDOWS_PLATFORM else ':'
```
Comment (aanand) [inline compose/const.py]: I think we can use [`os.pathsep`](https://docs.python.org/2/library/os.html#os.pathsep) here?

> The character conventionally used by the operating system to separate search path components (as in `PATH`), such as `':'` for POSIX or `';'` for Windows. Also available via `os.path`.

Comment (dnephin) [inline compose/const.py]: Oh nice, yes definitely. I couldn't find this earlier.

**Comment by dnephin** [comment]
Now using `os.pathsep`

**Comment by dnephin** [comment]
Test failure is unrelated

**Comment by aanand** [comment]
LGTM

**Pull request merged**